firebase-docs.md

```diff
@@ -0,0 +1,113 @@
+## Table of Contents
+
+- [Installation](#installation)
+  - [Prerequisites](#prerequisites)
+- [Initialize Firebase](#initialize-firebase)
+  - [File Structure](#file-structure)
+
+# Installation
+
+## Prerequisites
+
+Before you start setting up Firebase and pushing dummy data, make sure you have the following prerequisites:
+
+1. **Google Account**: You need a Google account to create and manage Firebase projects.
+
+2. **Python 3 Environment**: Ensure you have Python 3 installed on your machine.
+
+3. **Firebase Project**: Create a Firebase project with the Realtime Database enabled. You can set up a project in the [Firebase Console](https://console.firebase.google.com/).
+
+4. **Firebase Admin SDK**: Install the Firebase Admin SDK in your project using `pip`:
+
+   ```bash
+   pip install firebase-admin
```

Comment:
I believe it would be more preferable to import the json data using the GUI ,as provided in Realtime Database section too. (Open to debate on that :smile: ) Compared to running in the script